lib/types/later.d.ts

```diff
@@ -101,7 +106,511 @@ declare module '@breejs/later' {
     [timeperiodAndModifierName: string]: number[] | undefined;
   }
 
-  const later: { parse: { text: (s: string) => ScheduleData } };
+  interface ParseStatic {
+    /**
+     * Create a recurrence builder for building schedule data.
+     */
+    recur(): RecurrenceBuilder;
+
+    /**
+     * Create schedule data by parsing a cron string
+     *
+     * @param [input] - A string value to parse.
+     * @param [hasSeconds] - Whether the cron expression has second part.
+     */
+    cron(input?: string, hasSeconds?: boolean): ScheduleData;
+
+    /**
+     * Create schedule data by paring a human readable string.
```

Comment:
```suggestion
     * Create schedule data by parsing a human readable string.
```

Reply:
I think there's a typo here?

Also if this is code you grabbed from somewhere upstream, they should get a PR to fix this typo in their code as well?